OpenRA.Mods.Common/Irc.cs

```diff
@@ -0,0 +1,21 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Threading;
+using Meebey.SmartIrc4net;
+
+namespace OpenRA.Mods.Common
+{
+	public static class Irc
+	{
+		public static IrcClient Client;
```

Comment:
I looked through for other thread safety issues - and actually the comment I made above using the channels as an example was the only one I could see.

The library doesn't seem to provide a thread-safe way to query ```JoinedChannels``` - but in reality I doubt we'll ever actually run into this problem so I can live with it.